scipy/cluster/hierarchy.py

```diff
@@ -1497,6 +1490,70 @@ def to_tree(Z, rd=False):
         return nd
 
 
+def to_newick_tree(Z, leaf_names):
```

Comment:
Maybe we should consider something like this?
```suggestion
def to_newick_tree(Z, /, leaf_names):
```